.env.template.development

```diff
@@ -0,0 +1,129 @@
+IN_DOCKER=true
```

Comment:
New template file - since the existing one is geared toward production use and isn't really good for development boxes